proto/paladin.proto

```diff
@@ -47,13 +47,29 @@ message PaladinTalents {
 
 message RetributionPaladin {
     message Rotation {
-		bool consecration = 1;
+		enum ConsecrateRank {
+			None = 0;
+			Rank1 = 1;
+			Rank4 = 2;
+			Rank6 = 3;
+		}
+		ConsecrateRank consecrateRank = 1;
+		bool exorcism = 2;
     }
     Rotation rotation = 1;
 
     PaladinTalents talents = 2;
     
     message Options {
+		enum Judgement {
+			None = 0;
+			Wisdom = 1;
+			Crusader = 2;
+		};
+		Judgement judgement = 1;
+		int32 csDelay = 2;
```

Comment:
Could we add unit names where appropriate for these fields too? So perhaps crusader_strike_delay_ms and haste_leeway_ms.